README.md

```diff
@@ -53,16 +53,17 @@ Currently, the only difference between the two is how long the `make-genesis` co
 
 ### Generating the genesis file
 
-Before running the node, you must first generate the genesis file. The contents of the genesis file are currently hardcoded in Rust, but we intend to make these configurable shortly. The genesis block currently sets up 2 accounts: a faucet account for a `POL` token, as well as a wallet account.
+Before running the node, you must first generate the genesis file. The contents of the genesis file are fully configurable through a genesis inputs file written in TOML. An example genesis inputs file can be found here: [genesis.toml](node/genesis.toml)
+
 
 To generate the genesis file, run:
 ```sh
 miden-node make-genesis
 ```
 
-This will generate 3 files in the current directory: 
+By default this will generate 1 file and 1 folder in the current directory: 
 - `genesis.dat`: the genesis file.
-- `faucet.fsk` and `wallet.fsk`: the public/private keys of the faucet and wallet accounts, respectively.
+- `accounts/` folder containing an `account{i}.mac` file for each provided account in the genesis inputs file: a serialised `AccountData` struct containing the account, account-seed, auth-scheme and auth-seed. Learn more: https://github.com/0xPolygonMiden/miden-base/blob/341f692edcb375c0473d8a8c19e2c5944c7c9b57/objects/src/accounts/data.rs#L17-L30
```

Comment:
minor nit: I'd use "directory" instead of "folder".

For accounts, I probably wouldn't make a reference to the code. Instead we could say something like: 

`accounts` directory containing `.mac` files (one per account) for the accounts defined in the genesis inputs file. Each `.mac` file contains full serialization of an account, including code, storage, and authentication info.